### DISCUSSION TITLE
How to correct date format to non-US

### DISCUSSION THREAD

**Discussion by andrewnicols** [facebook/docusaurus#7086] [Q&A] [answered]
I'm looking at various options, including the `showLastUpdateTime` option: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#showLastUpdateTime

However, when this is set to true, the documentation is printed with the US date format, which is not used in the rest of the world.

I can't see any docs on changing this to use the correct date format. Is it possible?

**Comment by Josh-Cena** [comment, marked as answer]
The reason is that the date is formatted according to the current locale, and the default locale defaults to `en`, and `en` is interpreted as `en-US`. For now, the only solution is to use `defaultLocale: "en-GB"` or something similar in your [site config](https://docusaurus.io/docs/i18n/tutorial#site-configuration) (you don't actually have to go through the rest, just setting i18n config is fine). In a future release, #6430 will allow you to set a custom calendar format without changing your locale. I still recommend the former, though, because it's semantically more correct: if you are not using the US format, your site is probably not facing US audience either, so you'd better use a locale other than `en` which is `en-US`.

**Reply by andrewnicols** [reply]
Thanks,

I guess I was expecting that the subtag would be picked up from browser headers - i.e that `en` would have a default of `en-US` but if the browser requested `en-AU` that relevant strings would be translated where possible without additional configuration.

Even if I do make multiple locales available it seems that the default locale is still used unless the language menu is implemented and the value selected. I would expect it to be fetched from the `navigator.language` API if possible.

Is there anything in the roadmap for this?

Thanks

**Reply by slorber** [reply]
`navigator.language` is a browser API. When building static files we don't know the language of your user. We must hardcode the final date string in the HTML files, at build time.
